openwisp_radius/tests/test_api.py

```diff
@@ -180,7 +180,7 @@ def test_api_password_change(self):
         }
         response = client.post(password_change_url, data=new_password_payload)
         self.assertEqual(response.status_code, 400)
-        self.assertIn("The two password fields didn't match.", response.data['new_password2'])
+        self.assertIn("The two password fields didn’t match.", response.data['new_password2'])
```

Comment:
try `str(response.data['new_password2'])`

Reply:
This one is passing on my machine 
`test_radius_group_delete_default_by_non_superuser` is the one which is failing after installing `django-freeradius` again. Local environment was not updated with GitHub repository.

Reply:
In last line, shouldn't it be `confirm_response` instead of `response` ?

Reply:
Django 2.2 uses `'` for contractions while Django 3 uses `’` for same. This is causing test to fail.